src/db.ts

```diff
@@ -218,6 +216,8 @@ export class Db {
    * Create a new collection on a server with the specified options. Use this to create capped collections.
    * More information about command options available at https://www.mongodb.com/docs/manual/reference/command/create/
    *
+   * Collection namespace validation occurs at operation time
```

Comment:
This is fine, but perhaps a bit confusing. `createCollection` is an operation against the server, besides the error type returned there is not much of a distinction between the driver or the server validating the namespace. This sentence would fit better on `db.collection()` which is _not_ an operation against the sever so there will be no error until an operation is performed. If we want to keep a mention here about when validation is performed I would change this sentence to say "validation is performed server-side", rather than "operation time" as this method is operation time.

Reply:
@nbbeeken I was thinking something more verbose such as "Collection namespace validation is performed server-side once an collection operation is attempted," or does "Collection namespace validation is performed server-side" capture the same information?

Reply:
I'll also be sure to add the db.collection API information in my next commit.

Reply:
> Collection namespace validation is performed server-side once an collection operation is attempted.

`an collection` -> `a collection`

But I think you can just say "Collection namespace validation is performed server-side" because `createCollection` is an operation, and it is kind of a database one. There is not really a concept of database vs. collection operations precisely but a collection is created inside a database so logically it is not really a collection operation.

Reply:
Sounds good, I'll change it the simpler wording.